voluncheer/profiles/models.py

```diff
@@ -5,13 +5,15 @@
 
 class UserType(models.IntegerChoices):
     """Container for all possible user types."""
+
```

Comment:
Is this suggested by the linter? I usually try to avoid unnecessary white space.

Reply:
Again, flask8 autocorrect